v1/content.yaml

```diff
@@ -161,7 +163,7 @@ paths:
           type: string
       produces:
         - text/html; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/HTML/1.6.0"
-        - application/json
+        - application/problem+json
```

Comment:
Note this is a replace. I don't think Parsoid would produce regular `application/json` here.